pom.xml

```diff
@@ -2371,6 +2371,11 @@ under the License.
 								<exclude>org.apache.flink.util.function.SerializableFunction</exclude>
 								<exclude>org.apache.flink.util.function.SupplierWithException</exclude>
 								<exclude>org.apache.flink.util.function.ThrowingConsumer</exclude>
+								<!-- Disable check for two two sink methods. -->
```

Comment:
😅  Yep it is. I'll fix it; is there anything else?

Reply:
A question about `DiscardingSink`, `PrintSinkFunction`, `RichSinkFunction`
should they also be marked as deprecated?

I'm asking since `SinkFunction` is marked as deprecated and those 3 classes implement `SinkFunction`

Reply:
I think it makes sense to also mark those as deprecated

Reply:
all others are good to me

Reply:
I've updated the PR; I'll squash the commits when merging, but added them as separate commits to this PR